ReleaseNotes/eng/Altibase_Migration_Center_7_14_Release_Notes.md

```diff
@@ -0,0 +1,264 @@
+
+<br/>
+
+## 2.2 Bug-Fixes
+
+### BUG-50263	Data Type Mapping Update: BINARY_DOUBLE of Oracle, TimesTen, and Tibero is Now Mapped to Altibase DOUBLE, Replacing the Previous VARCHAR Mapping
+
+The BINARY_DOUBLE types of Oracle, TimesTen, and Tibero are compatible with the Altibase DOUBLE type, so the default data mapping type need to be changed. However, Oracle, TimesTen, and Tibero support special values of NaN(Not a Number) and INF(Infinity), while Altibase does not. For these values, data loss can occur.
+
+### BUG-50821	Altibase to Oracle Migration Feature is No Longer Supported
+
+Altibase to Oracle migration feature is no longer provided in accordance with company policy.
+
+### BUG-50827 	Data Type Mapping Update: The TimesTen Binary Type is Now Mapped to Altibase BYTE, Replacing the Previous BLOB Mapping
+
+The default data mapping type of TimesTen Binary type is more suitable for BYTE type than Altibase BLOB type, which is a large binary data type, considering data size range and compatibility. Therefore, the data mapping type of TimesTen Binary type is changed from Altibase BLOB type to BYTE type.
+
+### BUG-51034	Provide Empty String Data Conversion Function for Migration to Altibase
+
+Since Altibase does not distinguish between Empty String and Null data, unexpected results may occur when migrating records with Empty String. To prevent this, a function of converting Empty String data into a form suitable for Altibase is added to the Migration Option.
+
+### BUG-51035	Provide Conversion Function for Not Null & Default ''(Empty String) Columns During Migration to Altibase
+
+Since Altibase does not distinguish between Empty String and Null data, columns with Not Null and Default ''(Empty String) settings in other DBMS cannot be migrated as they are. To solve this problem, the Migration Option provides the function of converting Not Null & Default ''(Empty String) column into DDL suitable for Altibase.
+
+### BUG-51075	Migration Center UI Enhancement: Add Scrollbar and Adjust Height for Improved Usability in Option Window
+
+As options are continuously added to the Migration Center, the length of the existing Option window is too long, which may cause inconvenience to use. Add a scrollbar to the Option window and modify the height of the Option window.
+
+### BUG-51076	Migration Center UI Enhancement: Allow User-Adjustable Separation Bar in the Main Window
+
+In some cases, the Project tree and DB Properties window on the left side of the Migration Center main window may not be visible at certain resolutions. In the left window, the user cannot use the Migration Center normally in this case because the user cannot arbitrarily adjust the size. Modify the Project tree and DB Properties window on the left side of the main window to allow the user to adjust the size.
```

Comment:
```suggestion
In some cases, the Project tree and DB Properties window on the left side of the Migration Center main window may not be visible at certain resolutions. In the left window, the user cannot normally use the Migration Center in this case because the user cannot arbitrarily adjust the size. Modify the Project tree and DB Properties window on the left side of the main window to allow the user to adjust the size.
```

그냥 어순이 이게 자연스러워서요 ㅎㅎ

Reply:
> 그냥 어순이 이게 자연스러워서요 ㅎㅎ

수정하였습니다 ㅎㅎ 감사합니다.